server/test/unit/server/agent/direct/test_agent.py

```diff
@@ -32,10 +32,25 @@ class NotFound(Exception):
     pass
 
 
+class Context(Mock):
```

Comment:
This class name doesn't really tell me much. Consider using a name that isn't so generic.